### ISSUE TITLE
[Improvement request] Critical dependency: the request of a dependency is an expression

### ISSUE THREAD

**Issue by matthewlilley** [graphprotocol/graph-client#303]
We use the generated SDK within multiple Next.js applications, and are seeing a lot of these messages. 

![Screenshot_62](https://user-images.githubusercontent.com/6063350/197873080-802468bb-4f83-48c1-b713-b8f038c1e536.png)

How can these be avoided?

**Comment by ardatan** [comment]
Are they preventing you to build the client? If no, you can ignore it.

**Comment by matthewlilley** [comment]
@ardatan No, not at all. It's more of a nausiance. It clogs up terminal so makes debugging a little difficult.

**Comment by matthewlilley** [comment]
Wondered how we can mute it.

**Comment by ardatan** [comment]
It depends on your bundler. You can check the bundler's documentation for that.

**Comment by ardatan** [comment]
Ok I'm reopening it but this is not a bug definitely. This has 0 effect on runtime or it doesn't block you from building the project. 
Maybe we'll find a way to get rid of these false warnings from webpack.

**Comment by pugbyte** [comment]
I have the same issue, does anyone know how to configure Webpack so that this warning goes away?

**Comment by LilaRest** [comment]
Same issue here, and also interested in webpack configuration :)
